.github/workflows/main.yml

```diff
@@ -137,14 +143,22 @@ jobs:
 
   test:
     needs: build
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os || 'ubuntu-latest' }}
     strategy:
       fail-fast: false
       matrix:
         include:
           - browser: Chrome1280x1024
           - browser: FirefoxTouch
           - browser: FirefoxNoTouch
+          - browser: Chrome1280x1024
+            os: windows-2019
+          - browser: Chrome1280x1024
+            os: macos-latest
+          - browser: FirefoxRetina
+            os: ubuntu-latest
```

Comment:
SInce `ubuntu-latest` is the default, I think this can be dropped.
```suggestion
```

Another minor nit, can we move this up into the list of Firefox browsers?